src/charts/trends/trends-chart.component.js

```diff
@@ -101,7 +102,7 @@
          </div>
        </div>
        <div class="row">
-         <div class="col-md-6">
+         <div class="col-md-4">
            <form role="form"">
```

Comment:
oh the double quote quote... was already there, could fix if you want, happened to me too... might want to do a search on the page for any remaining ones